### PR TITLE
Update Denmark holidays: remove "Store bededag" starting 2024

### PR DESCRIPTION
**Pull request by malthe**
## Proposed change

See https://www.ft.dk/samling/20222/lovforslag/l13/index.htm.

## Type of change

- [ ] New country holidays support (thank you!)
- [x] Supported country holidays update (calendar discrepancy fix, localization)
- [ ] Existing code/test/process improvement (best practice, cleanup, refactoring, optimization)
- [ ] Dependency upgrade (version update)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (a code change causing existing functionality to break)
- [ ] New feature (new python-holidays functionality in general)

## Checklist

- [x ] I've followed the [contributing guidelines][contributing-guidelines]
- [x ] This PR is filed against `beta` branch of the repository
- [x ] This PR doesn't contain any merge conflicts and has clean commit history
- [x ] The code style looks good (`make pre-commit`)
- [x ] I've added tests to verify that the new code works and all tests pass locally (`make test`, `make tox`)
- [x ] The related [documentation][docs] has been added/updated (check off the box for free if no updates is required)

[contributing-guidelines]: https://github.com/dr-prodigy/python-holidays/blob/beta/CONTRIBUTING.rst
[docs]: https://github.com/dr-prodigy/python-holidays/tree/beta/docs/source

### CODE REVIEW

File: holidays/countries/denmark.py [hunk 1]
```diff
@@ -39,7 +39,9 @@ def _populate(self, year):
         self[easter_date + td(days=-2)] = self.tr("Langfredag")
         self[easter_date] = self.tr("Påskedag")
         self[easter_date + td(days=+1)] = self.tr("Anden påskedag")
-        self[easter_date + td(days=+26)] = self.tr("Store bededag")
+        # See https://www.ft.dk/samling/20222/lovforslag/l13/index.htm
+        if year < 2024:
```
Comment (KJhellico) [inline holidays/countries/denmark.py:43]:
```suggestion
        if year <= 2023:
```
Just for code style consistency.

**Comment by KJhellico** [comment]
LGTM

**Comment by arkid15r** [comment]
Hi @malthe,
thanks for updating this!

**Comment by arkid15r** [comment]
Merged into beta!

**Pull request merged**